packages/common/src/tables/user_terms/mod.rs

```diff
@@ -0,0 +1,20 @@
+use bdk::prelude::*;
+
+#[derive(validator::Validate)]
+#[api_model(base = "/v1/user_terms", table = user_terms)]
```

Comment:
```suggestion
#[api_model(base = "/v1/user-terms", table = user_terms)]
```

use kebab-convetion in url.
it's RFC standard.